en/components/select.md

```diff
@@ -6,79 +6,77 @@ _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angu
 
 ## Select Demo
 
-The Select is built by combining an input containing the current selection with a dropdown that appears to let you select an item.
+The Select is built by combining an input containing the current selection with a drop down that appears to let you select an item.
 
 <img class="responsive-img" src="../images/select_demo.png" srcset="../images/select_demo@2x.png 2x" />
 
 ## Input Type
 
-The Select supports the same three input types typical for the [Input](input.md) component: a line type for a more airy style, border type for a more structured perception on solid color backgrounds, and a boxed type that is most appropriate when placed on top of a vivid image to improve readability.
+The Select supports the same two input types typical for the [Input](input.md) component: border type for a more structured perception on solid color backgrounds, and a boxed type that is most appropriate when placed on top of a vivid image to improve readability.
+
 
-<img class="responsive-img" src="../images/select_line.png" srcset="../images/select_line@2x.png 2x" />
-`line`
 <img class="responsive-img" src="../images/select_border.png" srcset="../images/select_border@2x.png 2x" />
-`border`
 <img class="responsive-img" src="../images/select_box.png" srcset="../images/select_box@2x.png 2x" />
-`box`
 
-## Interaction State
+## Enabled and Disabled States
 
-Each of the three Select input types above can be inserted either in an enabled or a disabled state.
+In Sketch and Adobe Xd, each of these presets can be inserted in an enabled or disabled state. In Figma on the other hand, by default the input types are always enabled and they can be switched to disable state form the property panel. 
 
 <img class="responsive-img" src="../images/select_enabledstate.png" srcset="../images/select_enabledstate@2x.png 2x" />
 <img class="responsive-img" src="../images/select_disabledstate.png" srcset="../images/select_disabledstate@2x.png 2x" />
 
-## State
+## States
+
+When the user interacts with the Select, its input goes through various selection states: **Idle** with a placeholder in the place of the content, **Filled** that shows once the user has finished adding content and moved on, **Focused** with a visual indication that the user is currently inserting a value.
 
-When the user interacts with the Select, its input goes through various selection states: **idle** with a placeholder in the place of the content, focused when the dropdown is open, filled once the user has finished with selection and moved on. These flexibility enhancements afford a more dynamic interaction design that can seamlessly flow into high-fidelity prototyping.
 
 <img class="responsive-img" src="../images/select_idle.png" srcset="../images/select_idle@2x.png 2x" />
```

Comment:
Maybe we should either reorder the images or reorder the states in the description, in order to be consistent.